src/csharp/K4a.sln

```diff
@@ -1,13 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28809.33
+# Visual Studio 15
```

Comment:
Is there a way that we can tell Visual Studio to not fight over this...

Reply:
This is a [bug with Visual Studio 2019](https://developercommunity.visualstudio.com/content/problem/516621/visual-studio-2019-is-still-changing-version-in-so.html) . There was a similar bug in [Visual Studio 2017] (https://developercommunity.visualstudio.com/content/problem/181684/visual-studio-writes-version-in-solution-file.html) that was fixed, so hopefully it gets fixed again. I believe this should list the lowest supported version, i.e. "Visual Studio 15".